bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml

```diff
@@ -4,6 +4,17 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-type id="battery-voltage">
```

Comment:
Consider if this should be marked as advanced:
```suggestion
	<channel-type id="battery-voltage" advanced="true">
```

Reply:
I would not use advanced here, as it is a meaningful channel that should also be displayed directly to the user

Reply:
OK.